packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -4420,7 +4420,7 @@ YoEdncuy+GQGzE9yLOhC4HNfHQXpqp2tMPdRlw==</ds:X509Certificate>
 
 <ApiOperation method="post" url="/api/v1/apps/${applicationId}/credentials/csrs" />
 
-Generates a new key pair and returns the Certificate Signing Request for it
+Generates a new key pair and returns the CSR (Certificate Signing Request) for it. The information in a CSR is used to verify and create your SSL certificate. It also contains the public key that will be included in your certificate.
```

Comment:
Term goes first and then the abbreviation in parens. Reworded second sentence a bit.


Generates a new key pair and returns the Certificate Signing Request (CSR) for it. The information in a CSR is used to verify and create your SSL certificate. It also contains the public key that is included in your certificate.

Reply:
Addressed this.